<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramView</class>
 <widget class="QWidget" name="HistogramView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QwtPlot" name="chartWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image Scaling</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="imageMinIntensity"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="imageMaxIntensity"/>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="imageScaleBox">
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sine</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Square Root</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Logarithmic</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QxtSpanSlider" name="scalingSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header location="global">qxt/qxtspanslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
